libsync/SyncMaster.cpp

```diff
@@ -36,22 +36,22 @@ static unsigned const c_maxSendTransactions = 128;
 
 void SyncMaster::printSyncInfo()
 {
-    SYNC_LOG(TRACE) << "[Sync Info] --------------------------------------------" << endl;
-    SYNC_LOG(TRACE) << "            IsSyncing:    " << isSyncing() << endl;
-    SYNC_LOG(TRACE) << "            Block number: " << m_blockChain->number() << endl;
+    SYNC_LOG(TRACE) << "[Sync Info] --------------------------------------------";
+    SYNC_LOG(TRACE) << "            IsSyncing:    " << isSyncing();
+    SYNC_LOG(TRACE) << "            Block number: " << m_blockChain->number();
     SYNC_LOG(TRACE) << "            Block hash:   "
-                    << m_blockChain->numberHash(m_blockChain->number()) << endl;
-    SYNC_LOG(TRACE) << "            Genesis hash: " << m_syncStatus->genesisHash << endl;
+                    << m_blockChain->numberHash(m_blockChain->number());
+    SYNC_LOG(TRACE) << "            Genesis hash: " << m_syncStatus->genesisHash.abridged();
```

Comment:
could we change the log to one line to avoid multi threads unordered log?

Reply:
done